### DISCUSSION TITLE
Audio muted after changing audio source volume

### DISCUSSION THREAD

**Discussion by iotregenesis** [pschatzmann/ESP32-A2DP#257] [Q&A] [answered]
Hi all,

I've been using this library about 4 weeks now. Not really a fast progress since I am new to all this bluetooth and A1S.
I manage to build A2DP sink example with SSP authorization. This library is awesome. Thank you Phil Schatzmann.

The are 2 problems I find now:

1.  every time I change the audio quality at the source (my android phone) -- be it the volume or the equalizer -- suddenly the audio at A1S audio kit is gone. The bluetooth connection is still there "connected". I have to reset the board, unpair the device from my android, and then start the connection from the beginning. If I did not restart the board there will be message on my android saying "Can't communicate with Audiokit" when I tried to pair them.
2. This problem did not immediately happen if I change the volume at the sink side (A1S audio kit board). But after maybe 10 minutes or so suddenly the audio was muted too.

Could anyone help me understand what had happened, what could be the cause or what should I do to fix that?

Cheers,
Fahmy

**Comment by pschatzmann** [comment, marked as answer]
I suggest that you try with the latest commit of the A2DP library. 

I think that this might be related to new i2S queue. I have moved the new queue to a separate class so that the old functionality is working as before...
